res/lang/bossbetty/bossbetty_pt-br.txt

```diff
@@ -78,34 +78,34 @@ BETTY_PRESENT_9            = Anel
 BETTY_PRESENT_10           = Iate
 
 
-#AWARDNAME_CULTURE            = 
-#AWARDNAME_NEWS               = 
-#AWARDNAME_AUDIENCE           = 
-#AWARDNAME_CUSTOMPRODUCTION   = 
-#AWARD_WON                    = 
-#YOU_WON_THE_AWARDNAME        = 
-#PLAYERNAME_WON_THE_AWARDNAME = 
-#NOBODY_WON_THE_AWARDNAME     = 
-#CHANNEL_IMAGE                = 
-#FOR_X_HOURS                  = 
-#FOR_X_HOUR                   = 
-#ATTRACTIVITY                 = 
+AWARDNAME_CULTURE            = Sammy "pela Cultura"
```

Comment:
quotation marks - here it is `" ... "` - use the same across the localization please.

Reply:
he didn't change the quotation marks from the existing languages, so not his fault.

it's the same in all the other language files too.

we'd have to change and harmonize the quotation marks for every language then.

Reply:
yes, other localisations might need harmonization too ...

BUT ... not all quotation marks are "in use" in other languages. So double quots are I guess non-English, the sidewards looking arrows are more a "Swiss thing" but also known in German.

At the end quotation marks should be "unified" on a "per language" base.